source/index.html.erb

```diff
@@ -21,10 +21,12 @@ title: Index Page Example
           <p class="hero__description">
             Install GovWifi so your users can stay connected in any building.
           </p>
-          <%= link_to "Connect to GovWifi now", '/troubleshooting.html', class: "hero-button" %>
           <a href="https://www.gov.uk/guidance/set-up-govwifi-on-your-infrastructure" role="button" class="hero-button">
             Get GovWifi for your organisation
           </a>
+          <p class="hero__user-text">
```

Comment:
Were we discouraged from styling the second option as a button? I worry as text it gets lost in the page's content, but I know we've been back and forth on it.

Should we say "end user" rather than just "user"? Is that clearer?